pleiades/vocabularies/configure.zcml

```diff
@@ -72,6 +72,22 @@
     handler=".setuphandlers.remove_old_time_periods"
     />
 
+  <genericsetup:upgradeSteps
+      profile="pleiades.vocabularies:default"
+      source="0003"
+      destination="0004">
+    <genericsetup:upgradeDepends
+      title="Install relationship types registry vocabulary"
+      description=""
+      import_steps="plone.app.registry controlpanel"
```

Comment:
Are these necessary? There is no new control panel, and AFAICT no changes to registry.xml.